src/views/LandingPage/LandingPage.js

```diff
@@ -62,7 +62,7 @@ function LandingPage(props) {
 	let timeline = (
 		<div>
 			{width < minWidth && <Timeline width={'85%'} />}
```

Comment:
This is a timeline tag for smaller devices and is conditionally rendered. This also has to be commented out